onnxruntime/core/providers/nnapi/nnapi_builtin/builders/model_builder.cc

```diff
@@ -575,31 +545,71 @@ Status ModelBuilder::Compile(std::unique_ptr<Model>& model) {
       // and cpu is not in the target devices list
       return ORT_MAKE_STATUS(ONNXRUNTIME, EP_FAIL,
                              "The model cannot run using current set of target devices, ",
-                             nnapi_target_devices_detail_);
+                             GetDeviceDescription(nnapi_target_devices_));
+
     } else {
       use_create_for_devices = true;
     }
   }
 
+#ifndef NDEBUG
+  if (nnapi_target_devices_.size() > 1 && nnapi_target_devices_.back().type == ANEURALNETWORKS_DEVICE_CPU) {
+    auto* supported_ops = supported_ops_holder.get();
+    RETURN_STATUS_ON_ERROR_WITH_NOTE(
+        nnapi_.ANeuralNetworksModel_getSupportedOperationsForDevices(
+            nnapi_model_->model_, device_handles.data(),
+            static_cast<uint32_t>(device_handles.size() - 1), supported_ops),
+        "on getSupportedOperationsForDevices");
+
+    std::unordered_map<std::string, int32_t> optype_support;
+    const auto& node_indices = graph_viewer_.GetNodesInTopologicalOrder();
+    for (size_t idx = 0; idx < node_indices.size(); idx++) {
+      auto node_idx = node_indices[idx];
+      const auto* node(graph_viewer_.GetNode(node_idx));
+      if (!supported_ops[idx]) {
+        optype_support[node->OpType()]++;
```

Comment:
It was only built for DEBUG mode, so I think this log is necessary to show even more info  .

Reply:
yes, I see how this would be useful info to show in DEBUG mode.

I was wondering if we need to keep track of both an unsupported count and a supported count for a single op type. e.g., if there is an Add with 3 supported instances and 3 unsupported instances, then the current single count would be 0. is such a situation possible?